app/javascript/single-file-upload.js

```diff
@@ -0,0 +1,101 @@
+window.addEventListener("DOMContentLoaded", (event) => {
+  const fileInput = document.querySelector('.govuk-file-upload');
+  const maxFileSize = fileInput.dataset.maxSize;
+  const errorHeading = fileInput.dataset.errorHeading;
+  const errorMessage = fileInput.dataset.sizeErrorMessage;
+  const feedback = $(".single-file-upload__message");
```

Comment:
Don't think this needs to be a jquery call

Reply:
agreed, will update that

Reply:
updated